Provide FT_CALLBACK_DEF macro removed from
<freetype/config/ftconfig.h> in freetype 2.10.3
https://trac.macports.org/ticket/61306

--- base/fapi_ft.c.orig
+++ base/fapi_ft.c
@@ -124,6 +124,14 @@
 static void
 delete_inc_int_info(gs_fapi_server * a_server,
                     FT_IncrementalRec * a_inc_int_info);
+
+#ifndef FT_CALLBACK_DEF
+#ifdef __cplusplus
+#define FT_CALLBACK_DEF( x )  extern "C"  x
+#else
+#define FT_CALLBACK_DEF( x )  static  x
+#endif
+#endif
 
 FT_CALLBACK_DEF(void *)
 FF_alloc(FT_Memory memory, long size)
